integration-tests/fhir/src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#
+# Quarkus
+#
+quarkus.ssl.native=true
+#
+# Quarkus :: Camel
+#
+quarkus.camel.disable-jaxb=false
+#
+# Quarkus :: Camel :: FHIR
+#
+camel.fhir.enable-dstu2=false
+camel.fhir.enable-dstu3=true
+camel.fhir.enable-r4=false
```

Comment:
I still wonder why `dstu2` and `r4` are disabled. Could you please add a comment explaining it?

Reply:
Ah yeah I added the comment in the extension's doc but I'll add it here too

Reply:
Comment added, rebased on master, xml-common -> core-xml, thanks !

Reply:
Thanks for the explanation. I gathered some test runtime stats:

dstu3 only:   0:13 JVM, 06:31 native
all three on: 0:16 JVM, really long

I'd say the difference in the JVM mode is negligible. Could we perhaps run all three in the JVM mode and only dstu3 in the JVM mode?

Reply:
Something like this would do the trick:

https://github.com/ppalaga/camel-quarkus/commit/84b446650fc6a253458122342f09bb263eef5355#diff-70f5e932b22cdd19d8f1479077fc10d1R77-R86

Reply:
enabled all tests in JVM mode